Add eventfd

tkp1n opened the pull request.

tmds commented on src/Sources/linux.common/eventfd.cs:
```diff
@@ -0,0 +1,15 @@
+    public static partial class LibC
+    {
+        public static int EFD_CLOEXEC => 0x80000;
+        public static int EFD_NONBLOCK => 0x800;
```
Can you map these to `=> O_CLOEXEC` and `=> O_NONBLOCK`?

tmds commented:
Thanks for adding this. I'll be publishing a 0.4.0 in near future to nuget.org which will include this and the io_uring updates you've contributed.

tkp1n commented:
Addressed your comment, thanks for the review! Should be good to merge now.

> I'll be publishing a 0.4.0 in near future 

Looking forward to this! :shipit:
